tests/connect/standalone-x509-0001.phpt

```diff
@@ -1,5 +1,5 @@
 --TEST--
-Connect to MongoDB with using SSL and X.509 auth
+Connect to MongoDB with using SSL and X.509 auth (stream context)
```

Comment:
@bjori: please confirm that these title changes make sense. Best I could tell, tests 0002 and 0004 are analogous to 0001 and 0003, respectively, except they SSL context options passed directly to `$driverOptions` and processed by `php_phongo_populate_default_ssl_ctx()`, which only deals with a subset of the full context options found in 0001 and 0003.